packages/dd-trace/test/opentracing/propagation/text_map.spec.js

```diff
@@ -67,25 +68,60 @@ describe('TextMapPropagator', () => {
       expect(carrier).to.have.property('x-datadog-trace-id', '123')
       expect(carrier).to.have.property('x-datadog-parent-id', '456')
       expect(carrier).to.have.property('ot-baggage-foo', 'bar')
+      expect(carrier).to.have.property('baggage', 'foo=bar')
     })
 
     it('should handle non-string values', () => {
       const carrier = {}
-      const spanContext = createContext({
-        baggageItems: {
-          number: 1.23,
-          bool: true,
-          array: ['foo', 'bar'],
-          object: {}
-        }
-      })
+      const baggageItems = {
+        number: 1.23,
+        bool: true,
+        array: ['foo', 'bar'],
+        object: {}
+      }
+      const spanContext = createContext({ baggageItems })
 
       propagator.inject(spanContext, carrier)
 
       expect(carrier['ot-baggage-number']).to.equal('1.23')
       expect(carrier['ot-baggage-bool']).to.equal('true')
       expect(carrier['ot-baggage-array']).to.equal('foo,bar')
       expect(carrier['ot-baggage-object']).to.equal('[object Object]')
+      expect(carrier.baggage).to.be.equal('number=1.23,bool=true,array=foo%2Cbar,object=%5Bobject%20Object%5D')
```

Comment:
`array=foo%2Cbar` here decodes to
```
key:array
value:foo,bar
```

Which we should be able to handle correctly, depite the comma in `foo,bar`. I need to add a test with commas to make sure we handle this correctly .NET, thanks!